src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6062,6 +6062,11 @@ private bool IsUsingAliasInScope(string name)
                         }
                     default:
                         {
+                            // Ensure that there is an error thrown when an Unbound Generic is found.
+                            if (right is GenericNameSyntax genericNameRight && rightHasTypeArguments && typeArgumentsSyntax.Any(SyntaxKind.OmittedTypeArgument) && !IsUnboundTypeAllowed(genericNameRight))
+                            {
+                                diagnostics.Add(ErrorCode.ERR_UnexpectedUnboundGenericName, genericNameRight.Location);
+                            }
```

Comment:
Nit: newline

Reply:
Broke the line at a reasonable point to improve readability.